### PR TITLE
Lvnod 70 create modal messenger

### PR DESCRIPTION
**Pull request by yurii-kokhalevych**
## JIRA

* [Main JIRA ticket](https://ssu-jira.softserveinc.com/browse/LVNOD-70)
* [JIRA subtask](https://ssu-jira.softserveinc.com/browse/LVNOD-XXX)

## Code reviewers

- [x] @denisbodnar
- [x] @shyyka
- [x] @yuliannakorol

### Second Level Review

- [ ] @lhalam
- [ ] @4asof4ik

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo
## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=90%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: app/src/containers/Home/Home.js [hunk 1]
```diff
@@ -2,15 +2,17 @@ import React from 'react';
...
 export default class Home extends React.Component {
   render() {
     
     return (
       <Grid>
+      <Message />
         <Row className="show-grid">
           <Col sm={12} md={8}>
-            <PageHeader className="text-center">Da-Mi</PageHeader>
+            <PageHeader className="text-center">Da-Mi</PageHeader>
```
Comment (4asof4ik) [inline app/src/containers/Home/Home.js]: ??

File: app/src/containers/Message/Message.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  SIGNUP: 'You have successfully signed up! For confirmation please visit your e-mail'
+};
+
+export default class Message extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {show: false};
+  }
+
+  showModal = (e) => {
```
Comment (4asof4ik) [inline app/src/containers/Message/Message.js]: Check other pr and write single handler

File: app/src/containers/Message/Message.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { Modal, Button } from 'react-bootstrap';
+import style from './Message.scss';
+
+const MESSAGES = {
+  SIGNUP: 'You have successfully signed up! For confirmation please visit your e-mail'
```
Comment (4asof4ik) [inline app/src/containers/Message/Message.js]: Create messages file for all messages ask guys

**Review by 4asof4ik — CHANGES_REQUESTED** [review]

File: app/src/containers/Message/Message.scss [hunk 4]
```diff
@@ -0,0 +1,34 @@
+$bor-type: solid;
```
Comment (4asof4ik) [inline app/src/containers/Message/Message.scss]: create file common  styles and import into your styles

**Review by 4asof4ik — CHANGES_REQUESTED** [review]

**Review by denisbodnar — DISMISSED** [review]

**Review by shyyka — DISMISSED** [review]

**Review by yuliannakorol — DISMISSED** [review]

**Review by shyyka — DISMISSED** [review]

**Review by yuliannakorol — DISMISSED** [review]

File: app/src/containers/Message/Message.js [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+import style from './Message.scss';
+import messages from '../../helper/messages.js';
+
+export default class Message extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {show: false};
+  }
+
+  showModal = () => {
```
Comment (4asof4ik) [inline app/src/containers/Message/Message.js]: Where you use this ?

**Review by 4asof4ik — CHANGES_REQUESTED** [review]

File: app/src/containers/Message/Message.js [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+  render() {
+    return (
+      <Modal
+        show={this.state.show}
+        onHide={this.hideModal}
+        id="modal-container"
+      >
+        <Modal.Header closeButton>
+          <Modal.Title id="contained-modal-title-lg">
+            <p>{messages.signup}</p>
```
Comment (4asof4ik) [inline app/src/containers/Message/Message.js]: Need dynamic changing messages

**Review by 4asof4ik — CHANGES_REQUESTED** [review]

**Review by shyyka — DISMISSED** [review]

**Review by shyyka — DISMISSED** [review]

File: app/src/components/Message/Message.js [hunk 7]
```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { Modal, Button } from 'react-bootstrap';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import * as showActions from '../../redux/Signup';
+import style from './Message.scss';
+
+class Message extends React.Component {
+  hide = () =>{
```
Comment (denisbodnar) [inline app/src/components/Message/Message.js]: whitespace before {

**Review by denisbodnar — DISMISSED** [review]

**Review by denisbodnar — DISMISSED** [review]

**Review by yuliannakorol — DISMISSED** [review]

**Review by shyyka — DISMISSED** [review]

File: app/src/common.scss [hunk 8]
```diff
@@ -0,0 +1,20 @@
+$border-type-solid: solid;
```
Comment (4asof4ik) [inline app/src/common.scss]: example:
$border-bold: 10px solid $red;

**Review by 4asof4ik — CHANGES_REQUESTED** [review]

**Review by shyyka — DISMISSED** [review]

**Review by denisbodnar — DISMISSED** [review]

**Review by shyyka — DISMISSED** [review]

**Review by yuliannakorol — DISMISSED** [review]

File: app/src/components/Message/Message.js [hunk 9]
```diff
@@ -0,0 +1,46 @@
...
+    this.props.actions.showModal(false);
+  }
+
+  render() {
+    return (
+      <Modal
+        show={this.props.show}
+        onHide={this.hide}
+        id="modal-container"
+      >
```
Comment (4asof4ik) [inline app/src/components/Message/Message.js]: ?

**Review by 4asof4ik — CHANGES_REQUESTED** [review]

File: app/src/common.scss [hunk 10]
```diff
@@ -0,0 +1,14 @@
+$green: #008000;
+$red: #FF0000;
+$mint: #98ff98;
+$black: #000000;
+$white: #FFFFFF;
+$lightseagreen: #20B2AA;
+$borderTypeBase: solid;
+$borderWidthBase: 2px;
+$borderRadiusBase: 5px;
```
Comment (4asof4ik) [inline app/src/common.scss]: Please create normal var for border

**Review by 4asof4ik — CHANGES_REQUESTED** [review]

File: app/src/common.scss [hunk 11]
```diff
@@ -0,0 +1,8 @@
+$white: #ffffff;
+$green-dark: #008000;
+$green-ligth: #b3ffb3;
+$mint: #98ff98;
+$border-radius: 5px;
+$padding: 10px;
+$border1: 2px solid $white;
```
Comment (4asof4ik) [inline app/src/common.scss]: border-white maybe?

File: app/src/components/Message/Message.scss [hunk 12]
```diff
@@ -0,0 +1,28 @@
+@import '../../common.scss';
+
+$width: 400px;
+$left: 30%;
+$top: 10%;
+$position: absolute;
```
Comment (4asof4ik) [inline app/src/components/Message/Message.scss]: drop this

File: app/src/components/Message/Message.scss [hunk 13]
```diff
@@ -0,0 +1,28 @@
+@import '../../common.scss';
+
+$width: 400px;
+$left: 30%;
```
Comment (4asof4ik) [inline app/src/components/Message/Message.scss]: % bad think

File: app/src/components/Message/Message.scss [hunk 14]
```diff
@@ -0,0 +1,28 @@
+@import '../../common.scss';
+
+$width: 400px;
```
Comment (4asof4ik) [inline app/src/components/Message/Message.scss]: drop all your var

**Review by 4asof4ik — CHANGES_REQUESTED** [review]

**Review by denisbodnar — DISMISSED** [review]

**Review by yuliannakorol — DISMISSED** [review]

**Review by shyyka — DISMISSED** [review]

File: app/src/components/Message/Message.js [hunk 15]
```diff
@@ -0,0 +1,38 @@
...
+          <Modal.Title id="contained-modal-title-lg">
+            <p>{this.props.message}</p>
+          </Modal.Title>
+        </Modal.Header>     
+      </Modal>
+    );
+  } 
+}
+
+const mapStatetoProps = (state, ownProps) => ({
```
Comment (4asof4ik) [inline app/src/components/Message/Message.js]: where you use ownProps

File: app/src/common.scss [hunk 16]
```diff
@@ -0,0 +1,9 @@
+$white: #ffffff;
+$green-dark: #008000;
+$green-ligth: #b3ffb3;
+$mint: #98ff98;
+$border-radius: 5px;
+$padding: 10px;
+$border-white: 2px solid $white;
+$border-green-dark: 2px solid $green-dark;
+$margin: auto;
```
Comment (4asof4ik) [inline app/src/common.scss]: drop

**Review by 4asof4ik — CHANGES_REQUESTED** [review]

**Review by denisbodnar — DISMISSED** [review]

**Review by shyyka — DISMISSED** [review]

**Review by yuliannakorol — DISMISSED** [review]

**Review by shyyka — DISMISSED** [review]

**Review by denisbodnar — DISMISSED** [review]

**Review by yuliannakorol — DISMISSED** [review]

**Review by denisbodnar — APPROVED** [review]

**Review by yuliannakorol — APPROVED** [review]

**Review by shyyka — APPROVED** [review]

**Review by 4asof4ik — APPROVED** [review]

**Pull request merged**